### PR TITLE
Contribution7

### PR DESCRIPTION
**Pull request by mariamadronah**

### CODE REVIEW

File: asv/repo.py [hunk 1]
```diff
@@ -1,12 +1,9 @@
-# -*- coding: utf-8 -*-
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-
 import datetime
 
 from . import util
 
-
```
Comment (datapythonista) [inline asv/repo.py:9]: This one is needed. In short, the standard is that classes and functions should have two blank lines to separate them, the rest should be separated maximum by one. This is the full standard: https://www.python.org/dev/peps/pep-0008/

File: test/conftest.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import os
 import contextlib
 
-
```
Comment (datapythonista) [inline test/conftest.py:4]: This is the other one also needed.

**Review by datapythonista — COMMENTED** [review]
Thanks @mariamadronah, really nice clean up. There are just couple of blank lines that need to be restored, other than that this looks perfect.

**Comment by mariamadronah** [comment]
thanks a lot @datapythonista for the help through out me solving this issue its been great learning

**Pull request merged**